fastrtps.repos

```diff
@@ -6,16 +6,16 @@ repositories:
     fastcdr:
         type: git
         url: https://github.com/eProsima/Fast-CDR.git
-        version: v1.0.27
+        version: 1.0.x
     fastrtps:
         type: git
         url: https://github.com/eProsima/Fast-DDS.git
         version: 2.10.x
     fastddsgen:
         type: git
         url: https://github.com/eProsima/Fast-DDS-Gen.git
-        version: v2.4.0
+        version: 2.4.x
```

Comment:
We decided to support `2.5.x`. See #4479

```suggestion
        version: 2.5.x
```

Reply:
Ok great!